packages/core/src/modules/basic-messages/BasicMessagesModule.ts

```diff
@@ -46,6 +52,14 @@ export class BasicMessagesModule {
     dispatcher.registerHandler(new BasicMessageHandler(this.basicMessageService))
   }
 
+  private registerFeatures(featureRegistry: FeatureRegistry) {
+    featureRegistry.register<Protocol>(
```

Comment:
```suggestion
    featureRegistry.register(
```

This can be inferred by the TSEngine so we do not have to supply it all the time.